exercises/primzahlen.py

```diff
@@ -0,0 +1,10 @@
+start = 2
+end = 100
+  
+for i in range(start, end+1): 
+  if i>1: 
```

Comment:
Da `start = 2`, ist `i > 1` immer wahr, und somit überflüssig ;)

Hier gilt: expliziter ist "einfacher zu verstehen" - da der Startwert wohl nicht unbedingt verändert wird, wäre folgendes leichter verständlich:

```suggestion
for i in range(2, end+1): 
```